### ISSUE TITLE
Does it need another float() ?

### ISSUE THREAD

**Issue by neur0nic** [alilja/ratebeer#44]
I have a similar problem like in Issue #35. When trying to` _populate()` the `get_beers()` from the brewery `<Brewery('/brewers/paulaner-brauerei-schorghuber/61/')>` I get following output:

```
File "/usr/lib/python3.6/site-packages/ratebeer/models.py", line 388, in get_beers
       beer.abv = float(abv)
ValueError: could not convert string to float: '-' 
```

Because there is already an exception handling for `beer.abv` in line 175-178 I just changed line 388 to `beer.abv = abv` and it works for me.

**Comment by parryc** [comment]
@neur0nic does the pull request https://github.com/alilja/ratebeer/pull/42 work for your situation?

**Comment by neur0nic** [comment]
@parryc just tested it and it works. Thanks

**Comment by OrganicIrradiation** [comment]
Fixed by PR #42
